doc/Makefile

```diff
@@ -30,11 +31,15 @@ all: aspect.tag manual.pdf
 	@perl manual/cookbooks/annotate.pl $< >$@
 	@for i in `seq 1 10`; do sed -i 's/{\([^!]*\)!\([^!]*\)!\([^!]*\)!\([^}]*\)}/{\1!\2!\3\/\4}/' $@; done
 
+# Generate changes.h from the bits in modules/changes/
+modules/changes.h: modules/changes/* modules/create_changes.sh modules/current_changes_*
```

Comment:
Are you sure about this? https://www.gnu.org/software/make/manual/html_node/Wildcards.html suggests wildcards in targets and prerequisites are expanded automatically, and I tested touching a random file in `modules/changes/` and it recreated changes.h. Maybe `make` did learn a thing or two over its (long) lifetime :smile:.